### PR TITLE
Implemented about button 

### PR DESCRIPTION
**Pull request by arpnakumari**

### CODE REVIEW

File: PowerUp/app/src/main/res/layout/activity_about.xml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            android:layout_gravity="center_horizontal"
+            android:fontFamily="cursive"
+            android:gravity="center"
+            android:text="@string/about"
+            android:textAlignment="center"
+            android:textSize="@dimen/powerup_size"
+            android:typeface="serif"
+            android:textStyle="bold"
+            android:foregroundGravity="bottom"/>
+
```
Comment (anubhakushwaha) [inline PowerUp/app/src/main/res/layout/activity_about.xml]: Don't hardcode,add in string.xml

File: PowerUp/app/src/main/res/layout/activity_about.xml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        <TextView
+            android:text="Powerup focuses and strive to help the teen girls to make mature decisions about their reproductive health.While you enjoy the game, It will let you choose your course of action through a series of scenes and questions.
+
+We came up with PowerUp to fill the gap created by the society and schools regarding sex education and its importance.Since this game can be downloaded and is available world wide, It will help us eliminate the limitation of reach of sexual education to just developed nation and societies.
+
```
Comment (anubhakushwaha) [inline PowerUp/app/src/main/res/layout/activity_about.xml]: Don't hardcode,add in string.xml

**Comment by anubhakushwaha** [comment]
@arpnakumari  I have commented on gci comment section as well , follow up :)

File: PowerUp/app/src/main/res/layout/activity_about.xml (hunk 2 above)
Comment (arpnakumari) [inline PowerUp/app/src/main/res/layout/activity_about.xml]: okey @anubhakushwaha

**Pull request closed without merging**